entries/f/fpmarkets.com.json

```diff
@@ -10,6 +10,12 @@
     "documentation": "https://www.youtube.com/watch?v=aEXDRisRtUs",
     "keywords": [
       "investing"
+    ],
+    "regions": [
+      "-be",
+      "-jp",
+      "-nz",
```

Comment:
Where do you see New Zealand in the list of restricted countries?
On their .com site the list of restricted countries include:
>the United States of America, Australia, Japan, Democratic People's Republic of Korea (North Korea), the Islamic Republic of Iran, Republic of Iraq and Belgium

And on their .eu site the list of restricted countries are:
>the United States of America, Israel, Japan, North Korea (DPRK) or Belgium

Reply:
New Zealand is not in the list of countries on [the account opening page](https://portal.fpmarkets.com/register).